### PR TITLE
Bulgarian Translation - QoL Improvements

### PR DESCRIPTION
**Pull request by RacerBG**
A few tweaks here and there...

Regards,
Georgi

### CODE REVIEW

File: resources/l10n/bg/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -237,722 +238,723 @@ msgstr "%1% намерен %2%."
...
 
-#: src/gui/html/js/translateStaticText.ts:744
+#: src/gui/html/js/translateStaticText.ts:614
 msgid ""
 "This appears to be the first time you have run LOOT v%s. Here are some tips "
 "to help you get started with the interface."
 msgstr ""
-"Изглежда това е първото стартиране на LOOT v%s. Ето някои съвети, които ще "
-"са Ви от полза."
+"Изглежда това е първото стартиране на LOOT %s. Ето някои съвети, които ще са "
```
Comment (sibir-ine) [inline resources/l10n/bg/LC_MESSAGES/loot.po:904]: @RacerBG Just FYI, the `v` here is an abbreviation of `version`, & LOOT's version is subbed into `%s`. Is there an equivalent of this abbreviation in Bulgarian that can be used here?

Comment (RacerBG) [reply resources/l10n/bg/LC_MESSAGES/loot.po:904]: @sibir-ine It can be translated like "в.".

Comment (sibir-ine) [reply resources/l10n/bg/LC_MESSAGES/loot.po:904]: It'd probably be best to go with the latter to match the formatting if the original text then.

Comment (RacerBG) [reply resources/l10n/bg/LC_MESSAGES/loot.po]: It should be good to go now. :)

Comment (sibir-ine) [reply resources/l10n/bg/LC_MESSAGES/loot.po]: Yup, the rest looks good to me.

**Pull request merged**